native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech/DeepSpeechModel.java

```diff
@@ -47,17 +47,35 @@ public void freeModel() {
     }
 
    /**
+    * @brief Enable decoding using an external scorer.
```

Comment:
Since we touch this, we might need to ensure we have up-to-date https://github.com/mozilla/DeepSpeech/tree/master/native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech_doc#javadoc-for-sphinx

Reply:
I ran the commend and no files were changed, is that expected?

Reply:
Seems like some of the generated files are tracked and some aren't?

Reply:
```
Untracked files:
  (use "git add <file>..." to include in what will be committed)
    native_client/java/jni/deepspeech_wrap.cpp
    native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech_doc/DeepSpeech_Error_Codes.java
    native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech_doc/SWIGTYPE_p_ModelState.java
    native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech_doc/SWIGTYPE_p_StreamingState.java
    native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech_doc/SWIGTYPE_p_p_ModelState.java
    native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech_doc/SWIGTYPE_p_p_StreamingState.java
    native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech_doc/impl.java
    native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech_doc/implJNI.java
```

Reply:
Yeah, that should be safe. We only need a few of them, the SWIGTYPE stuff are not really supposed to be manipulated by people.